templates/default/net/netadd.html

```diff
@@ -41,10 +41,21 @@ <H1>{$layout.pagetitle}</H1>
 		<TD class="bold nobr"><IMG src="img/mac.gif" alt=""> {trans("Interface:")}</TD>
 		<TD colspan="3">
 			<INPUT type="text" size="23" name="netadd[interface]" value="{$netadd.interface}" {tip text="Enter network interface (optional)" trigger="interface"}>
-			<span class="bold">{trans("vlan:")}</span>
-			<INPUT type="text" size="5" name="netadd[vlanid]" value="{if $netadd.vlanid}{$netadd.vlanid}{/if}" maxlength="4" {tip text="Enter vlan ID (optional)" trigger="vlanid"}>
-		</TD>
-	</TR>
+        </TD>
+    </TR>
+    <TR>
+        <TD class="bold">
+            {icon name="vlan" label="VLAN:"}
```

Comment:
Przestajemy używać dwukropków w etykietach. Niepotrzebne są - ktoś mi ostatnio na to zwrócił uwagę.

Reply:
Usunięty.